how-to-setup-synthetic-monitoring/unstyled.html

```diff
@@ -0,0 +1,71 @@
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Adding Testing & Synthetics</title>
+</head>
+<body>
+  <h1>Adding Testing & Synthetics</h1>
+  
+  <p>This tutorial will guide you through adding Testing & Synthetics using Grafana's plugin system.</p>
+  
+  <h2>Section 1: Accessing Plugins</h2>
+  <p>In this section, you'll navigate to the Plugins section in Grafana to manage and install new plugins.</p>
+  
+  <span id="access-plugins" class="interactive" data-targetaction="sequence" data-reftarget="span#access-plugins">
+    <ul>
+      <li class="interactive" 
+          data-targetaction="highlight" 
+          data-reftarget="a[data-testid='data-testid Nav menu item'][href='/admin']"
+          data-requirements="navmenu-open,exists-reftarget">
+        Open the main menu and click <strong>Administration</strong>
+      </li>
+      
+      <li class="interactive" 
+          data-targetaction="highlight" 
+          data-reftarget="a[data-testid='data-testid Nav menu item'][href='/admin/plugins']"
+          data-requirements="exists-reftarget">
+        Click <strong>Plugins and data</strong>
+      </li>
+      <li class="interactive" 
+      data-targetaction="highlight" 
+      data-reftarget="a[data-testid='data-testid Nav menu item'][href='/plugins']"
+      data-requirements="exists-reftarget">
+      Click <strong>Plugins</strong>
+    </li>
+    </ul>
+  </span>
+
+  <h2>Section 2: Installing Synthetic Monitoring</h2>
+  <p>Now, you will search for and install the Synthetic Monitoring plugin.</p>
+
+  <span id="install-synthetic-monitoring" class="interactive" data-targetaction="sequence" data-reftarget="span#install-synthetic-monitoring">
+    <ul>
+      <!-- TODO: the plugins search input does not have an id, so we need to use a different selector, in this case the placeholder text -->
+      <li class="interactive" 
+          data-targetaction="formfill" 
+          data-reftarget="input[placeholder='Search Grafana plugins']"
+          data-targetvalue="Synthetic Monitoring"
+          data-requirements="exists-reftarget">
+        Type "Synthetic Monitoring" in the search bar
```

Comment:
Here's a trick.... you have the option to define an extra "comment" that shows up when you highlight a box.

When you highlight the search box and type something into it, and the user clicks "Show Me", it will draw an orange highlight box around the input.   By using an extra span, you can cause text to pop up explaining the element that's being highlighted.

Here's an example:  https://github.com/grafana/interactive-tutorials/blob/main/rca-demo-v2/unstyled.html#L84